spec/acceptance/bundle_without_spec.rb

```diff
@@ -2,8 +2,9 @@
 
 require "spec_helper"
 
-RSpec.describe "Bundler without flag" do
-  it "passes --without flag to Bundler on install" do
+RSpec.describe "Bundle without group" do
+  it "config set --local without group is honored by Bundler" do
+    pending "config set --local without group support seems broken, see: https://github.com/rubygems/rubygems/issues/8518"
```

Comment:
Literally this is broken, reported bug to bundler:
https://github.com/rubygems/rubygems/issues/8518